src/model/game.py

```diff
@@ -33,6 +33,7 @@ class Game(QObject):
     """
     gameUpdated = pyqtSignal(object, object)
     liveReplayAvailable = pyqtSignal(object)
+    ladderMapReveal = pyqtSignal(object)
```

Comment:
None of this new code is related to actual model of game data (such as 'live replay for the game is available now' modeled by liveReplayAvailable), it's strictly a matter of how it's shown in the UI. No changes in the model class should be necessary.

You can put this code in some separate listener class that will determine if info should be hidden based on a game, and schedule any timers and signals there.

Reply:
So no changes to game.py, the signal should come from gameset.py newLiveGame ?
I try to do it your way .. if I do it my way, you won't like it... so hint me. .. please.

Reply:
What we want to achieve here is add a logic of 'if we shouldn't display detailed game info right now, let's not do it, but schedule a state update later'. We could encapsulate this logic in some new 'Sensitive game info checker' class.

An instance of it could have a 'check if the game has (recent) sensitive info' method that takes a game object. Then we can add an 'emit a signal when this game's info can be shown' method that also takes a game object (and that can be cancelled or reset at a game update, or when a game is replaced by a different one). These methods can be called in chatter on every game update, for example.

This will put all 'when and how to delay sensitive game info' logic in one continguous chunk of code, which I think is the cleanest solution.